spec/unit/classes/server/database_ini_spec.rb

```diff
@@ -0,0 +1,140 @@
+require 'spec_helper'
```

Comment:
There was no coverage here, so I've added default coverage+tests for this particular change.